api/users/views.py

```diff
@@ -401,8 +405,48 @@ def get_default_odm_query(self):
             Q('is_registration', 'eq', True) &
             Q('contributors', 'eq', user._id)
         )
+
         permission_query = Q('is_public', 'eq', True)
         if not current_user.is_anonymous():
             permission_query = (permission_query | Q('contributors', 'eq', current_user._id))
         query = query & permission_query
         return query
+
+class UserNodeLogs(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin, UserMixin):
+
+    serializer_class = NodeLogSerializer
+    pagination_class = UserNodeLogPagination
+    view_category = 'users'
+    view_name = 'node_logs'
+
+    ordering = ('-date', )
+
+    def get_default_odm_query(self):
+        user = self.get_user()
+        if user.is_anonymous():
+            auth = Auth(None)
+        else:
+            auth = Auth(user)
+        raw_nodes = Node.find(
+            Q('contributors', 'eq', user) &
+            Q('is_folder', 'ne', True) &
+            Q('is_deleted', 'ne', True)
```

Comment:
Question: I haven't been keeping up with all API changes, but what is the `Bookmarks` node for?  Do we want logs related to the Bookmarks to show up here?  Because this is a node that seems to have been automatically created for me.